granulate_utils/containers/client.py

```diff
@@ -46,14 +46,29 @@ def __init__(self) -> None:
         if self._docker_client is None and self._cri_client is None:
             raise NoContainerRuntimesError()
 
-    def list_containers(self, all_info: bool = False) -> List[Container]:
+    def list_containers(self, all_info: bool = False, running: bool = False) -> List[Container]:
```

Comment:
I think that `only_running` is a better name for this parameter (which has the meaning of "If True, only running containers are returned.").

Reply:
Same for the respective docker/cri functions